### ISSUE TITLE
Some ads not blocked on some sites

### ISSUE THREAD

**Issue by JnTon** [bromite/bromite#1411]
### Bromite version

Version: `92.0.4515.176`
Arch: `arm64`
Android version: `11`
Device model: `sagit`

### Flags

These are the flags changed under `chrome://flags`:

No change

### Is this bug about the SystemWebView?

No

### Is the bug reproducible with latest version?

Yes

### Can the bug be reproduced with corresponding Chromium version?

No

### Is this bug happening in an incognito tab?
Yes

### Allow ads for the site; is the bug caused by the adblocker?
Yes

### Is the bug a crash?

No

### Describe the bug

Basically ads don't get blocked. I see ads on website such as YouTube (either in standard and incognito tab).

### Steps to reproduce the bug

Just going to a random page that have ads, like a YouTube video.

### Expected behavior

Ads get blocked.
### Screenshots

![IMG_20210924_012207_066](https://user-images.githubusercontent.com/84038748/134596747-7f79025b-93c9-4602-a021-822115058ce8.jpg)

**Comment by csagan5** [comment]
This has been discussed extensively, you can find the issues and discussion by searching; is it just YouTube or also other sites which are affected?

**Comment by JnTon** [comment]
@csagan5 Twitch is affected too. I also saw ads on other sites, but now I don't remember exactly what they were (it seems to me that they were some newspaper-websites).

**Comment by ygz213** [comment]
I can confirm this. (`TCL-8079` - `arm` - `5.0.1`)

**Comment by csagan5** [comment]
Please connect to the websites affected using [remote Dev Tools](https://developer.chrome.com/docs/devtools/remote-debugging/) and compare the blocked network requests using a desktop version with uBlock Origin.

Can you identify what are the requests that are blocked on desktop uBlock origin and not in Bromite?

**Comment by ygz213** [comment]
93.0.4577.83 fixed the problem for me.

**Comment by csagan5** [comment]
That is unlikely; it is probably a problem with the filters that are not covering perfectly a certain region, or due to some missing advanced feature. I will close the issue if nobody can provide for the difference in network requests explaining what is not blocked.

**Comment by JnTon** [comment]
@csagan5 I'll do it, give me only the time to get a new cable, btw on `93.0.4577.83` the problem is still not fixed for me.

**Comment by Guiltynine** [comment]
Ads are showing up on apkmirror.com while using this version..

**Comment by JnTon** [comment]
@csagan5 since Chrome was not working on my PC, I used Brave: I've turned off every type of blocking made by Brave in the settings and I've installed uBlock Origin. I took the uBlock Origin log and then I've removed any script blocked also blocked by Bromite (_dev tools=>Network=>Blocked Requests_; with all the websites in _"Desktop site_" mode).
I also discovered that Twitch isn't blocked by Ublock Origin either, but I found these solutions that could be implemented into Bromite too: https://github.com/pixeltris/TwitchAdSolutions

The following are the requests not blocked by Bromite but blocked by Ublock Origin on desktop:

CanYouBlockIt - Extreme Test ([tested page](https://canyoublockit.com/extreme-test/)): https://cryptpad.fr/code/#/2/code/view/Pa0Eewl1IC1Qvq2fMPcMz8VxTn+u2f7jWmbdlBiQjxk/

YouTube ([tested page](https://www.youtube.com/watch?v=HeQX2HjkcNo)): https://cryptpad.fr/code/#/2/code/view/aRo6n-ALaWlw86gZPg1-3gCXC-Rh-6fRiV3mQtJ1jRQ/

ApkMirror ([tested page](https://www.apkmirror.com/apk/t-mobile-usa/t-mobile-digits/t-mobile-digits-2-7-0-release/t-mobile-digits-2-7-0-android-apk-download/)): https://cryptpad.fr/code/#/2/code/view/negiPLAHqk3LoQhI4Aby1ZSs6d7P-G6-iugBR+FHyyg/

TheHackerNews ([tested page](https://thehackernews.com/2021/10/researchers-discover-uefi-bootkit.html?m=1)): https://cryptpad.fr/code/#/2/code/view/bSLFZZ-PQ47IcIqMd5MAr4WzdEMwgBX27gG8a17MpFI/

**Comment by nospace-here** [comment]
Latest version `95.0.4638.78` on `arm64` also has ads also an YouTube again. Previous version had blocked it.

**Comment by HonestAbe7** [comment]
Would it be possible to add Chrome extensions in the future or not? When I had Kiwi browser installed, before it didn't autofill addresses, I had Ghostery & Fair Adblocker as added ad blockers in case this browser missed any.

**Comment by Disqu3-mirOir-qt** [comment]
with new google manifest v3 many extension use csp capacity not alive for chromiun browser based many adblocker dead 2022-2023 ublock develloper said end extesion with manifest v3 

many work required for extension who dead

**Comment by nospace-here** [comment]
![Screenshot_20211229-152054](https://user-images.githubusercontent.com/77444261/147671708-c300fe44-6d9c-424c-a446-48dc2e79601c.png)
Link: https://www.n-tv.de/der_tag/Ampel-rechnet-nicht-mit-baldiger-Cannabis-Legalisierung-article23026300.html
Bromite version
Version: 96.0.4664.54
Arch: arm64
Android version: 12
Device model: sunfish
I only disabled jit on bromite

**Comment by VPnKEXXRxfqtQWYPllfVlqJXXSthDhZQzsyTuBB** [comment]
I think there is a problem with the adblocking engine, I have enabled the exact filters what bromite uses with firefox UBO and it fixed the issue

**Comment by maboxx** [comment]
Exactly the same for me. Fennec with UBO works on ntv.de, in Bromite the ads are displayed.

**Comment by vjrmy** [comment]
please @csagan5  add more filter list adblock bromite like 1hosts (lite), Oisd, anudeep's blacklist, and notracking. those are great filter list and enable cosmetic filters like Kiwi Browser which is Kiwi browser now open source

**Comment by mrawbabylon** [comment]
hi im having this problem on youtube too!!! i just re-installed this on my new phone so i have the newest version i think... can anyone explain to me in layman's terms how to fix this 😅

**Comment by MuntashirAkon** [comment]
For YouTube, you can use the following user script:

https://github.com/GSRHackZ/No-ADS-YouTube

**Comment by JustAnUserLol** [comment]
> For YouTube, you can use the following user script:
> 
> https://github.com/GSRHackZ/No-ADS-YouTube

It only skips ads, doesn't block them...
